packages/common/__tests__/server.test.ts

```diff
@@ -0,0 +1,34 @@
+import { createServer } from '../src'
+
+describe('CreateServer', () => {
+  it('Yoga accepts graphiql option in different formats', async () => {
+    const lazy = () => {
+      const graphiqlOptionAsFalse = createServer({
```

Comment:
Could you have this type checking test here instead?
https://github.com/klippx/graphql-yoga/blob/graphiql-type/packages/common/type-api-check.ts#L7

Reply:
Done!